pkg/app/utils.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+)
+
+// AppendRandString, appends a random string to the passed string value
+func AppendRandString(name string) string {
```

Comment:
Are we using this function outside the package? Please avoid exporting the function unless required.